test/checkpoint/test_checkpoint.h

```diff
@@ -45,7 +45,7 @@
  * of the other three types.
```

Comment:
Does this comment need updating if we removed on of the table types? I read the comment and it actually isn't clear. It almost feels like it wasn't updated in the past when it went to 4 table types and has been wrong. But more eyes should look at it.

Reply:
Though I have no idea what "special type of mixed" (i.e. `MIX`) means, it does seem [when the 4th type was introduced](https://github.com/wiredtiger/wiredtiger/commit/9b7021497f4c61dca5448778b5d78c1990c6a561#diff-23188d7b317796883e7b7d2350e4fa9c45c82baaef7957189c1fe1a4a2170abf) the comment was not updated. So now we are removing one type and it offsets the missing update :)